cmds/core/kill/kill_test.go

```diff
@@ -7,10 +7,19 @@ package main
 import (
 	"bytes"
 	"fmt"
+	"os/exec"
 	"strings"
 	"testing"
 )
 
+func getUnusedPID() string {
+	cmd := exec.Command("true")
+	cmd.Start()
```

Comment:
Start() returns an error and it should probably be checked (along with changing getUnusedPID to be a [test helper](production/network/regtest:multidiff_flipper) which helps clarify where the failing call is:

```
func getUnusedPID(t *testing.T) {
  t.Helper()
  ...
  if err := cmd.Start(); err != nil {
    t.Fatalf("failed to start `true` command: %v", err) 
  }
}
```